RELEASING.md

```diff
@@ -41,7 +41,14 @@ You should have no local changes and tests should be passing.
 
 1. Go to https://github.com/tigera/operator/milestones and create any new milestones that should exist (e.g., next patch release)
 
-## Creating a release on RH Catalog
+## Updates for new Calico CRDs
+
+If the release includes new Calico CRDs, add the new CRDs to `hack/gen-csv/get-manifests.sh`.
```

Comment:
Also the template file if I understand correctly.

Reply:
Thanks, updated. For some reason i thought I had automated that section of the template file.